packages/react-component-library/src/primitives/FloatingBox/FloatingBox.stories.tsx

```diff
@@ -13,29 +13,24 @@ export default {
   },
 } as Meta<FloatingBoxWithEmbeddedTargetProps>
 
-export const Default: Story<FloatingBoxWithEmbeddedTargetProps> = (props) => (
+const Template: Story<FloatingBoxWithEmbeddedTargetProps> = (props) => (
   <FloatingBox isVisible renderTarget={<div />} {...props}>
     <div style={{ padding: '0 1rem' }}>
       <pre>Arbitrary JSX content</pre>
     </div>
   </FloatingBox>
 )
 
+export const Default = Template.bind({})
 Default.args = {
   scheme: FLOATING_BOX_SCHEME.LIGHT,
+  'aria-label': 'A floating box',
```

Comment:
This error here was:

https://dequeuniversity.com/rules/axe/4.4/aria-dialog-name

This simply sets an accessible name for these stories (which can be done in multiple ways).